### PR TITLE
Fix: fixed wrong zod version in lock file

### PR DESCRIPTION
**Pull request by sethsandaru**
Fixed wrong zod version in lock file, from 3.22.2 to 3.8.2 again

### CODE REVIEW

**Review by chris-muller — CHANGES_REQUESTED** [review]
This needs to be a new release with a new version number or it wont be propagated to the NPM registry.

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {
     "name": "zod-http-schemas",
-    "version": "2.0.3",
+    "version": "2.0.1",
```
Comment (chris-muller) [inline package-lock.json:3]: The latest version is `2.0.3`, this should either match the package.json file or we should update all versions to 2.0.4 for the next release we'll do.

**Review by chris-muller — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (sethsandaru) [reply package-lock.json]: right thank you

**Comment by sethsandaru** [comment]
@Antman261 please help us to merge this one and release v2.0.4

**Review by tuanlc — APPROVED** [review]

**Pull request merged**